### PR TITLE
📝 Update README and Contributing Guide

### PR DESCRIPTION
**Pull request by frissyn**
the current readme sucks xD, fixed it 👍🏽

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -10,37 +10,46 @@ We recommend using the following tools:
 + Git Bash or GitHub Desktop as your **Version Control**.
 + TypeScript, Eslint and Prettier as your VSC **extensions**.
 
-You can also write code in [Replit](https://replit.com/@leon332157/repl-it-electron) however it is not recommended since it's not optimal for electron.
```
Comment (spotandjake) [inline .github/CONTRIBUTING.md:13]: We should probably make a new Repl that runs this and keep this here

Comment (frissyn) [reply .github/CONTRIBUTING.md:13]: Or we could just delete that line because deving this on replit is dumb 🤔

File: README.md [hunk 2]
```diff
@@ -1,51 +1,44 @@
 # replit-desktop [![Discord](https://img.shields.io/discord/437048931827056642.svg?logo=discord)](https://discord.gg/5gcPC6B) [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3bce49c376cf4c2bb1d2813d6b12dd6a)](https://www.codacy.com/manual/leon332157/replit-desktop?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=repl-it-discord/replit-desktop&amp;utm_campaign=Badge_Grade) [![Run on Replit](https://replit.com/badge/github/leon332157/replit-desktop)](https://replit.com/github/replit-discord/replit-desktop)
 
-
-### Join the [Replit discord](https://replit.com/discord) for quicker communication with our community and ask questions about the app.
-
-#### Before you make an issue, come in the discord and ask us as the issue could be fixed on another branch, which isn't always up-to-date with `dev`.
 ## Summary
```
Comment (spotandjake) [inline README.md:3]: We should add a note about the new replit pwa, and possible replish and the new replit pwa but we should note the differences between these. as the pwa would work better for some peope and replish would work better for others.

**Review by spotandjake — APPROVED** [review]
Looks fine a few minor changes noted in the comments might be smart but they are not needed at the moment.

File: README.md (hunk 2 above)
Comment (frissyn) [reply README.md:3]: I don't know anything about `replish` so you'd have to ask @leon332157 about it. As for the PWA I could add a notice under the summary or smth

**Review by leon332157 — APPROVED** [review]

**Comment by leon332157** [comment]
Might as well say that the app is deprecated.

**Comment by frissyn** [comment]
fr xD

**Comment by spotandjake** [comment]
> 
> 
> Might as well say that the app is deprecated.

I ddont know if we should add aaa deprectition warning as this adds eatures to the repli that are nott native up to you though. this app might still be preferable to the pwa to some people leon you may want to add a link to replish though as that may be more in tthe style of what people are looking for.

**Comment by frissyn** [comment]
@spotandjake bro where ur punctuation and grammar at? xD

**Pull request merged**